libobs/util/platform-nix.c

```diff
@@ -272,6 +274,62 @@ char *os_get_program_data_path_ptr(const char *name)
 	return str;
 }
 
+#if defined(__OpenBSD__)
+// a bit modified version of https://stackoverflow.com/a/31495527
+ssize_t os_openbsd_get_executable_path(char *epath)
+{
+	int mib[4];
+	char **argv;
+	size_t len;
+	const char *comm;
+	int ok = 0;
+
+	mib[0] = CTL_KERN;
+	mib[1] = KERN_PROC_ARGS;
+	mib[2] = getpid();
+	mib[3] = KERN_PROC_ARGV;
+
+	if (sysctl(mib, 4, NULL, &len, NULL, 0) < 0)
+		abort();
+
+	if (!(argv = malloc(len)))
```

Comment:
use bmalloc

Reply:
done.

Reply:
@kkartaltepe There wasn't any real reason to use bmalloc here. It's fine to use normal malloc/free pair in some cases, especially if it's solely within one scope and not being returned or something. The obs allocation functions are mostly for allocation tracking purposes.